lib/eslint/eslint.js

```diff
@@ -568,7 +573,7 @@ class ESLint {
         }
         const {
             filePath,
-            warnIgnored = false,
```

Comment:
Removing the inconsistent default, this will fall back to `processOptions` defaults